src/cow-react/modules/limitOrders/pure/Widget/index.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components/macro'
+
+export const Widget = styled.div`
+  background: white;
```

Comment:
can you make a minimal `fixture` for this component? 

Just exporting Widget as the `default` in an `index.cosmos.tsx` file would suffice